### PR TITLE
[ASSET-29] Add top100 tokens' information from BSC Etherscan

### PR DESCRIPTION
**Pull request by jormal**
# Pull Request

## Description

BSC의 [Etherscan](https://bscscan.com) 에서 top 100 token을 수집합니다.

Related issue: [ASSET-29](https://pi-lab.atlassian.net/browse/ASSET-29)

### Changes

- [`ecf6216`](https://github.com/bifrost-platform/asset-info-v2/pull/35/commits/ecf6216dea367b85217a9f0fe54a74212b48fec8): Tag released workflow 수정
- [`b2bd020`](https://github.com/bifrost-platform/asset-info-v2/pull/35/commits/b2bd020de4ae95231068800108559ef794591203): ListModel이 append와 sort 지원하도록 수정
- [`738b6d8`](https://github.com/bifrost-platform/asset-info-v2/pull/35/commits/738b6d84efaa9e2d474bf85336f89a797547c3a0): ID guide 추가
- [`b0f5861`](https://github.com/bifrost-platform/asset-info-v2/pull/35/commits/b0f58611d21ffaf86bf524fd51f0065058dbebfa): Top 1~50 추가
- [`50ff97b`](https://github.com/bifrost-platform/asset-info-v2/pull/35/commits/50ff97bbd98375e9a3acd33afb72fdb5c61c703d): Top 51~100 추가
- [`0a6503a`](https://github.com/bifrost-platform/asset-info-v2/pull/35/commits/0a6503a641d68c8f77c522a39e10c565e6930d61): Version 갱신

### Types of Changes (multiple options can be selected)

- [x] Create asset information
- [x] Update asset information
- [ ] Delete asset information
- [ ] Other `{{Please add description here}}`

## Checklist

- [x] Did you pass the tests? (Did you execute `pytest -l` in your PC and get a green light?)
- [x] Did you run the pre-process?
- [x] Have you added and run tests to validate the changes?
- [x] Have you updated the documentation?
- [x] Have you updated the version in `{PWD}/VERSION` if you need?

[ASSET-29]: https://pi-lab.atlassian.net/browse/ASSET-29?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: assets/flow-1/info.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  "id": "flow-1",
+  "images": {
+    "png128": true,
+    "png256": true,
+    "png32": true,
+    "png64": true,
+    "svg": true
+  },
+  "name": "FLOW",
+  "references": [],
```
Comment (yjkellyjoo) [inline assets/flow-1/info.json:23]: https://coinmarketcap.com/currencies/flow/
https://www.coingecko.com/en/coins/flow

File: assets/op-1/info.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  "id": "op-1",
+  "images": {
+    "png128": true,
+    "png256": true,
+    "png32": true,
+    "png64": true,
+    "svg": true
+  },
+  "name": "Optimism",
+  "references": [],
```
Comment (yjkellyjoo) [inline assets/op-1/info.json:23]: 아 `-1` 부터는 reference link 안 넣는건가요?

**Review by yjkellyjoo — APPROVED** [review]

File: assets/op-1/info.json (hunk 2 above)
Comment (jormal) [reply assets/op-1/info.json:23]: 아뇨 해당하는 reference가 없습니다. `op-0`와는 다른 coin으로 보여져서요

File: assets/flow-1/info.json (hunk 1 above)
Comment (jormal) [reply assets/flow-1/info.json:23]: flow도 마찬가지입니다.

Comment (jormal) [reply assets/flow-1/info.json:23]: 올려주신 주소들에서 BSC network의 `0xC943c5320B9c18C153d1e2d12cC3074bebfb31A2` 를 찾을 수 없었습니다.

Comment (jormal) [reply assets/flow-1/info.json:23]: 만약 해당 contract가 `flow-0`의 것과 동일하다면 정보를 `flow-0`으로 옮겨야합니다.

Comment (yjkellyjoo) [reply assets/flow-1/info.json:23]: https://bscscan.com/token/0xc943c5320b9c18c153d1e2d12cc3074bebfb31a2
image
BscScan에서 두 링크로 연결시켜주긴 하는데 이것만으로는 확실하지 못하다 보시는걸까요?

Comment (yjkellyjoo) [reply assets/flow-1/info.json:23]: image

https://etherscan.io/token/0x5c147e74D63B1D31AA3Fd78Eb229B65161983B2b

coinmarketcap에서는 wrapped flow 컨트랙트로 연결시키네요... 혼란하다

Comment (jormal) [reply assets/flow-1/info.json:23]: 네 binance의 일방적인 pegging이라고 간주했습니다..

**Pull request merged**